tests/unit/test_modulegraph/test_modulegraph.py

```diff
@@ -23,15 +23,15 @@
     from io import StringIO
 
 TESTDATA = os.path.join(
-        os.path.dirname(os.path.abspath(__file__)),
```

Comment:
I can't tell what you changed in this file. There are too many formatting changes obscuring the actual code changes.